### ISSUE TITLE
Can't change backgroundColor, textColor, Position

### ISSUE THREAD

**Issue by Anzyme** [triniwiz/nativescript-toasty#51]
Hi, I'm trying this plugin on a school project but can't manage to change backgroundColor, textColor, Position, ...
Nativescript : 7.1.2
Language : Angular
Platform : Android 11

Here is the code I'm using :
`const toast = new Toasty({ text: error, position: ToastPosition.TOP, backgroundColor: new Color('RED'), textColor: ('#FF0000') });
toast.show();`

I can change the text but it doesn't care about the colors or any argument I set.

**Comment by Anzyme** [comment]
Just tested on an emulator with Android 8 and everything works perfectly.
So I guess it was and issue with Android 11 all along :/

**Comment by triniwiz** [comment]
Yeah the behaviour changed https://developer.android.com/about/versions/11/behavior-changes-11#text-toast-api-changes

**Comment by fatrix39** [comment]
@triniwiz Are you going to solve this issue?

**Comment by triniwiz** [comment]
It's not possible on android 11+ using the out of box platform api since they restricted a couple things, if needed we will need to implement something custom

**Comment by fatrix39** [comment]
ok, thanks. I will implement my own component in this case
